src/System.Security.Cryptography.ProtectedData/pkg/System.Security.Cryptography.ProtectedData.pkgproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.ProtectedData.csproj">
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Security.Cryptography.ProtectedData.builds" />
+    
+    <ExcludeDefaultValidateFramework Include="netstandardapp1.5" />
+    <!-- Only supported on windows, so overriding
+         the validation RID's -->
+    <ValidateFramework Include="netstandardapp1.5">
+      <RuntimeIDs>win7-x86;win7-x64</RuntimeIDs>
```

Comment:
@ericstj Given the new auto-PNSE libraries (with your oh-so-colorful metaphor of a name :)) is this still the way to do this?